src/meta/meta.c

```diff
@@ -36,20 +36,17 @@
 
 int MetaData_ToJson_Internal(MetaData *metadata, JsonGenerator *gen)
 {
-    int rc;
+    int rc = 0;
```

Comment:
Why decide to change to implement in right way. In my suggest, I think the implement of left is better than the right one except  for changing to return  0 when satisfy with "!metadata->data" to keep align with the new right one.